src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +51,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = factory.openSession()) {
+            Query<Movie> getAllMoviesQuery = session.createQuery(
+                    "from Movie", Movie.class);
```

Comment:
In `CinemaHallDaoImpl` you've used constant for defining query. Be consistent with one approach, so move this one to the constant as well or keep all of them as local variables

Reply:
fixed